docs/extend/lifecycling-with-zwesvstc.md

```diff
@@ -12,86 +14,40 @@ To start Zowe, the script `<INSTANCE_DIR>/bin/zowe-start.sh` is run from a USS s
 
 When `run-zowe.sh` is run in the USS shell that `BPXBATSL` creates, it executes the file `<INSTANCE_DIR>/instance.env`.  This file sets a number of shell variables, such as `ROOT_DIR` that points to the directory with the `<RUNTIME_DIR>`, variables for all of the ports used by the Zowe components, and other configuration data. For more information, see [Reviewing the instance.env file](../user-guide/configure-instance-directory.md#reviewing-the-instance.env-file).
 
-## Zowe components
+**Note:**
+
+The scripts of core Zowe components and some extensions use the helper library `<RUNTIME_DIR>/scripts/utils`.  Currently, these are not publicly supported. Future releases of Zowe might provide these as supported system programming interfaces (SPIs) and include their usage in the Zowe documentation.  
```

Comment:
Is this supposed to point to `scripts/utils`, `bin/utils`, or both? I think nearly all of `scripts/utils` should be deleted in future, not added as supported SPI